Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -654,7 +663,7 @@ private void updateNavigationDrawerItems() {
         }
     }
 
-    public void launchEditTags(View view) {
+    public void launchEditTags() {
```

Comment:
Minor (🔍): Consider making this method `private` since it is no longer necessary to have this as `public`. I would even consider moving the whole `startActivity(...)` within the `onClick(...) { ... }` method above and remove this method altogether to simplify a bit.

Reply:
Good idea.  I moved the `startActivity(...)` statement to the `onClick(...)` listener in 0b483fef.

Reply:
Thanks for the refactor! 🌟